graph_tool/models.py

```diff
@@ -1,3 +1,5 @@
 from django.db import models
```

Comment:
a django model!!

Reply:
yes! so the file can be saved to local folder